.idea/workspace.xml

```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might be worth removing this file for now as this is your local config - can you add `.idea/` to `.gitignore` - it'll stop you needing to remember to not commit this every time.